packages/expo-media-library/android/src/test/java/expo/modules/medialibrary/TestUtils.kt

```diff
@@ -0,0 +1,47 @@
+package expo.modules.medialibrary
+
+import android.content.ContentResolver
+import android.content.Context
+import android.database.Cursor
+import io.mockk.every
+import io.mockk.mockk
+import org.robolectric.fakes.RoboCursor
+
+typealias CursorResults = Array<Array<out Any?>>
+
+fun mockCursor(cursorResults: CursorResults): Cursor {
+  val cursor = RoboCursor()
+  cursor.setColumnNames(MediaLibraryConstants.ASSET_PROJECTION.toMutableList())
+  cursor.setResults(cursorResults)
+  return cursor
+}
+
+fun mockContentResolver(cursor: Cursor?): ContentResolver {
+  val contentResolver = mockk<ContentResolver>()
+  every { contentResolver.query(
+    MediaLibraryConstants.EXTERNAL_CONTENT,
+    MediaLibraryConstants.ASSET_PROJECTION,
+    any(), any(), any())
+  } returns cursor
+  return contentResolver
+}
+
+fun mockContentResolverForResult(cursorResults: CursorResults)
+  = mockContentResolver(mockCursor(cursorResults))
```

Comment:
I think that the android studio will put it in one line 🤔

Reply:
If the line is too long, it automatically splits it to new line.